clients/drcachesim/tests/raw2trace_unit_tests.cpp

```diff
@@ -199,6 +199,17 @@ make_core()
     return entry;
 }
 
+offline_entry_t
+make_window_id()
```

Comment:
This should probably take in the id as otherwise it will not be possible to have 2 new window ids in one sequence.